packages/vite-plugin-checker/src/checkers/eslint/main.ts

```diff
@@ -91,7 +91,7 @@ const createDiagnostic: CreateDiagnostic<'eslint'> = (pluginConfig) => {
 
         if (terminal) {
           for (const d of diagnostics) {
-            consoleLog(diagnosticToTerminalLog(d, 'ESLint'))
+            consoleLog(diagnosticToTerminalLog(d, 'ESLint'), 'info')
```

Comment:
@danielroe I believe this should be either `error` or `warning`, depending on `d.level`. Now, the ESLint errors/warnings won't be logged if log level is higher than info.
![image](https://github.com/user-attachments/assets/353a8648-3d1a-4248-ad49-4557ad8343fc)